fast64_internal/mk64/mk64_actors.py

```diff
@@ -0,0 +1,200 @@
+                box.box().label(text="Children of this object will be wrapped in GEO_OPEN_NODE and GEO_CLOSE_NODE.")
+
+    def draw_behavior_params(self, obj: bpy.types.Object, parent_box: bpy.types.UILayout):
+        game_object = obj.fast64.mk64.game_object  # .bparams
+        parent_box.separator()
+        box = parent_box.box()
+        box.label(text="Behavior Parameters")
+
+        box.prop(game_object, "use_individual_params", text="Use Individual Behavior Params")
+
+        if game_object.use_individual_params:
+            individuals = box.box()
+            individuals.label(text="Individual Behavior Parameters")
+            column = individuals.column()
+            for i in range(1, 5):
+                row = column.row()
+                row.prop(game_object, f"bparam{i}", text=f"Param {i}")
+            individuals.separator(factor=0.25)
+            individuals.label(text=f"Result: {game_object.get_combined_bparams()}")
+        else:
+            box.separator(factor=0.5)
+            box.label(text="All Behavior Parameters")
+            box.prop(game_object, "bparams", text="")
+            parent_box.separator()
+
+    def draw(self, context):
+        prop_split(self.layout, context.scene, "gameEditorMode", "Game")
+        box = self.layout.box().column()
+        column = self.layout.box().column()  # added just for puppycam trigger importing
+        box.box().label(text="MK64 Object Inspector")
+        obj = context.object
+        prop_split(box, obj, "mk64_obj_type", "Object Type")
+        if obj.mk64_obj_type == "Actor":
+            prop_split(box, obj, "mk64_model_enum", "Model")
+            if obj.mk64_model_enum == "Custom":
+                prop_split(box, obj, "mk64_obj_model", "Model ID")
+            box.operator(SearchModelIDEnumOperator.bl_idname, icon="VIEWZOOM")
+            box.box().label(text="Model IDs defined in include/model_ids.h.")
+            prop_split(box, obj, "mk64_behaviour_enum", "Behaviour")
+            if obj.mk64_behaviour_enum == "Custom":
+                prop_split(box, obj, "mk64_obj_behaviour", "Behaviour Name")
+            box.operator(SearchBehaviourEnumOperator.bl_idname, icon="VIEWZOOM")
+            behaviourLabel = box.box()
+            behaviourLabel.label(text="Behaviours defined in include/behaviour_data.h.")
+            behaviourLabel.label(text="Actual contents in data/behaviour_data.c.")
+            self.draw_behavior_params(obj, box)
+            self.draw_acts(obj, box)
+
+        elif obj.mk64_obj_type == "None":
+            box.box().label(text="This can be used as an empty transform node in a geolayout hierarchy.")
+
+    def draw_acts(self, obj, layout):
+        layout.label(text="Acts")
+        acts = layout.row()
+        self.draw_act(obj, acts, 1)
+        self.draw_act(obj, acts, 2)
+        self.draw_act(obj, acts, 3)
+        self.draw_act(obj, acts, 4)
+        self.draw_act(obj, acts, 5)
+        self.draw_act(obj, acts, 6)
```

Comment:
```suggestion
        for i in range(1, 6):
            self.draw_act(obj, acts, i)
```
something like this is cleaner imo (idr if the range include 1 and 6 in my suggestion but I think it does)